### ISSUE TITLE
Google Contacts calendar (i.e., birthdays) not showing any events

### ISSUE THREAD

**Issue by tew42** [andstatus/todoagenda#141]
Since around the start of this year, the Google Contacts calendar events (i.e., birthdays) aren't showing up in the widget any more. The calendar itself is listed and activated. In the regular calendar app, events/birthdays show up just fine.

It's maybe worth mentioning that the issue appeared around the time Google introduced the internal service (un)linking mechanism for EU users, which can break the birthday calendar entirely when unlinking either Contacts or Calendar service. However I have both the relevant services linked, and as mentioned it works fine in the regular calendar app.

App version 4.9.0

**Comment by yvolk** [comment]
This issue is being created from time to time for many years, 
and the problem is NOT in the widget, but in the Google Calendar and/or Android System's or Account's settings...
Please try to use tips starting from #68

**Comment by tew42** [comment]
Thanks - I did already try complete un-syncinc and re-syncing in Account settings, clearing all calendar data, completely resetting my Google account login etc., and none of that had helped.

Your suggestion did however prompt me to check a few third-party calendar apps to see if the problem would persist with them also. Strangely, the issue got fixed in ToDo Agenda after setting up one of those (either Samsung or Komorabi Simple Calendar - impossible to tell now). Will have to see if ToDo Agenda is able to pull updates in the future now that I have removed the third-party calendar app again...
